docs/glossary.md

```diff
@@ -3,8 +3,8 @@
 
 **Authorization**
 :Kubernetes does not currently have an authorization system.  Anyone with the cluster password can do anything.  We plan
-to add sophisticated authorization, and to make it pluggable.  See the [access control design doc](./devel/access.md) and
-[this issue](https://github.com/GoogleCloudPlatform/kubernetes/issue/1430).
```

Comment:
This is a test comment
 * point 1
 * point 2 in *bold*? **bold?** ***boldbold***